java/java.lsp.server/vscode/syntaxes/java.tmLanguage.json

```diff
@@ -1571,42 +1582,45 @@
 		},
 		"strings": {
 			"patterns": [
+				{
+					"begin": "\"\"\"",
+					"beginCaptures": {
+						"0": {
+							"name": "punctuation.definition.string.begin.java"
+						}
+					},
+					"end": "\"\"\"",
+					"endCaptures": {
+						"0": {
+							"name": "punctuation.definition.string.end.java"
+						}
+					},
+					"name": "string.quoted.triple.java",
+					"patterns": [
+						{
+							"match": "(\\\\\"\"\")(?!\")|(\\\\.)",
+							"name": "constant.character.escape.java"
+						}
+					]
+				},
 				{
 					"begin": "\"",
 					"beginCaptures": {
 						"0": {
-							"name": "punctuation.definition.string.begin.java string.quoted.double.java"
+							"name": "punctuation.definition.string.begin.java"
 						}
 					},
 					"end": "\"",
 					"endCaptures": {
 						"0": {
-							"name": "punctuation.definition.string.end.java string.quoted.double.java"
+							"name": "punctuation.definition.string.end.java"
 						}
 					},
+					"name": "string.quoted.double.java",
 					"patterns": [
-						{
```

Comment:
Sorry, not sure about this removal. Please see:
https://github.com/apache/netbeans/pull/6745

Reply:
Injection created for string templates.